dcr/loader.go

```diff
@@ -69,10 +71,20 @@ func CreateWallet(ctx context.Context, params CreateWalletParams, recovery *Reco
 			if err != nil {
 				return nil, fmt.Errorf("unable to decrypt wallet seed: %v", err)
 			}
+			if len(wd.EncryptedSeedPass) != 0 {
+				encSeedPass, err := hex.DecodeString(wd.EncryptedSeedPass)
+				if err != nil {
+					return nil, fmt.Errorf("unable to decode encrypted seed pass: %v", err)
+				}
+				seedPass, err = DecryptData(encSeedPass, params.Pass)
+				if err != nil {
+					return nil, fmt.Errorf("unable to decrypt wallet seed pass: %v", err)
+				}
+			}
```

Comment:
there appears to be no way to specify the wallet seed password unless it exists encrypted in the recovery data.  Maybe that's intentional, but it doesn't seem all too useful to me (i would expect passwords to not need to be saved with the other seed data, and to only be provided as user input when needed).

Reply:
Yeah idk its this thing https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#from-mnemonic-to-seed

Looking again it seems to say we always apply the PBKDF2 even with no password. We cant do that with the 15 word seed but I guess we should with the new ones.